tests/models/Attribution.test.js

```diff
@@ -121,6 +121,26 @@ const test = async app => {
     await tap.ok(response instanceof Attribution)
   })
 
+  await tap.test('Get all attributions by publicationId', async () => {
+    let attributions = await Attribution.getAttributionByPubId(publication.id)
+
+    await tap.ok(attributions)
+    await tap.ok(attributions[0] instanceof Attribution)
+    await tap.equal(attributions[0].name, 'Jane Doe')
+  })
+
+  await tap.test(
+    'Delete attributions with a certain role of a given publication',
+    async () => {
+      let numDeleted = await Attribution.deleteAttributionOfPub(
+        publication.id,
+        'author'
+      )
+
+      await tap.ok(numDeleted > 0)
```

Comment:
You should check the exact number.
Before you do that, also create an attribution of type 'editor' and check that the number of attributions deleted does not include the editor.